.circleci/config.yml

```diff
@@ -21,11 +21,13 @@ commands:
   install-just:
     description: "Install just"
     steps:
-      - run: |
-          wget -qO - 'https://proget.makedeb.org/debian-feeds/prebuilt-mpr.pub' | gpg --dearmor | sudo tee /usr/share/keyrings/prebuilt-mpr-archive-keyring.gpg 1> /dev/null
-          echo "deb [arch=all,$(dpkg --print-architecture) signed-by=/usr/share/keyrings/prebuilt-mpr-archive-keyring.gpg] https://proget.makedeb.org prebuilt-mpr $(lsb_release -cs)" | sudo tee /etc/apt/sources.list.d/prebuilt-mpr.list
-          sudo apt update
-          sudo apt install just
+      - run:
+          name: "Install just"
```

Comment:
This just made the step title easier to read in circle ci (instead of the long wget command being the name)

Reply:
Good call